server/properties.lua

```diff
@@ -0,0 +1,228 @@
+local properties = {}
+local propertiesGroups = {}
+
+--- Create a new property
+---@param data table {name: string, interior: number, furnished: boolean, garage: boolean, coords: vector4, price: number, rent: number}
+---@return boolean
+local function createProperty(data)
+    if not data or not data.coords then return false end
+    local result = MySQL.Sync.fetchAll('SELECT id FROM properties ORDER BY id DESC LIMIT 1', {})
+    local id = result?[1].id + 1 or 1
+
+    local name = id .. ' ' .. data.name
+    exports.oxmysql:insert('INSERT INTO properties (name, interior, furnished, garage, coords, price, rent) VALUES (@name, @interior, @furnished, @garage, @coords, @price, @rent)', {
+        ['@name'] = name,
+        ['@interior'] = data.interior,
+        ['@furnished'] = data.furnished or false,
+        ['@garage'] = data.garage or false,
+        ['@coords'] = json.encode(data.coords),
+        ['@price'] = data.price,
+        ['@rent'] = data.rent,
+    })
+end
+
+local function fillInsideTable()

```

Comment:
function name is unclear

Reply:
I mean it fills the Inside table, with the players inside the property, could be populateInsideTable too i guess, this naming stuff is hard :sad:

Reply:
renamed